xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -1578,6 +1578,9 @@ You can use asynchronous read operations to avoid these dependencies and their d
   
 ## Remarks  
  If the value of the <xref:System.Diagnostics.ProcessStartInfo.StandardOutputEncoding%2A> property is `null`, the process uses the default standard output encoding for the standard output. The <xref:System.Diagnostics.ProcessStartInfo.StandardOutputEncoding%2A> property must be set before the process is started. Setting this property does not guarantee that the process will use the specified encoding. The application should be tested to determine which encodings the process supports.  
+
+> [!IMPORTANT]
+>  Use encodings that don't have BOM. For example use `new UnicodeEncoding(false, false)` rather than `Encoding.Unicode`.
```

Comment:
I think this remark without this issue as a context is very confusing.

Reply:
```suggestion
>  When interacting with applications that cannot handle BOM correctly consider using BOM-less encoding. For example use `new UnicodeEncoding(false, false)` rather than `Encoding.Unicode`.
```